doc/releases/changelog-dev.md

```diff
@@ -59,44 +56,102 @@
   
   mol = qml.qchem.Molecule(symbols, geometry, basis_name='sto-3g')
   core, one, two = qml.qchem.electron_integrals(mol)()
-  algo = DoubleFactorization(one, two)
   
-  # print the number of non-Clifford gates and logical qubits
-  print(algo.gates, algo.qubits)
+  algo = DoubleFactorization(one, two)
   ```
 
   ```pycon
+  >>> print(algo.gates, algo.qubits)
   103969925, 290
   ```
 
-  The methods of the `FirstQuantization` and the `DoubleFactorization` classes can be also accessed
-  individually. For instance, the logical qubits can be computed by providing the inputs needed for
-  this estimation without initiating the class. 
+  The methods of the `FirstQuantization` and the `DoubleFactorization` classes 
+  can be also accessed individually without initiating the class:
+
+  - The number of logical qubits with `qubit_cost`:
 
   ```python
   n = 100000
   eta = 156
   omega = 169.69608
   error = 0.01
-  qml.resource.FirstQuantization.qubit_cost(n, eta, omega, error)
   ```
   
   ```pycon
+  >>> qml.resource.FirstQuantization.qubit_cost(n, eta, omega, error)
   4377
   ```
 
-  In addition to the number of non-Clifford gates and logical qubits, some other quantities such as
-  the 1-norm of the Hamiltonian and double factorization of the second-quantized Hamiltonian can be
-  obtained either by initiating the classes or by directly calling the functions.
+  - The number of non-Clifford gates with `gate_cost` (TODO: check this):
+
+   ```python
+    n = 100000
+    eta = 156
+    omega = 169.69608
+    error = 0.01
+    ```
+    
+    ```pycon
+    >>> qml.resource.FirstQuantization.gate_cost(n, eta, omega, error)
+    TODO
+    ``` 
+
+<h4>Differentiable error mitigation 🧑‍🔧</h4>
+
+  TODO 
+
+* Differentiable zero-noise-extrapolation error mitigation is now available.
```

Comment:
This section needs a bit of a rework. @Qottmann knows already 😸

Reply:
This is already the reworked version. Could you specify what needs clarification and what is fine to link to other resources (elaborate doc-string and demo)? I dont know where to make the trade-off here between explaining ZNE from scratch but keeping it short and accessible

Reply:
Hey sorry! I skimmed past this thinking @Qottmann hadn't updated it yet 🤪. I'll read it!

Reply:
Looks great @Qottmann! Have some small changes that I'll push soon